pkg/addons/default/aws_node_test.go

```diff
@@ -46,15 +46,15 @@ var _ = Describe("default addons - aws-node", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(awsNode.Spec.Template.Spec.Containers).To(HaveLen(1))
 			Expect(awsNode.Spec.Template.Spec.Containers[0].Image).To(
-				Equal("602401143452.dkr.ecr.eu-west-1.amazonaws.com/amazon-k8s-cni:v1.4.1"),
+				Equal("602401143452.dkr.ecr.eu-west-1.amazonaws.com/amazon-k8s-cni:v1.5.7"),
 			)
 
 		})
 
-		It("can update 1.12 sample to latest", func() {
+		It("can update 1.14 sample to latest", func() {
 			rawClient.AssumeObjectsMissing = false
 
-			_, err := UpdateAWSNode(rawClient, "eu-west-2", false)
+			_, err := UpdateAWSNode(rawClient, "eu-west-1", false)
```

Comment:
why this change?

Reply:
this is for new file sample-1.14.json, which is the new file as part of this PR. I can modify the testdata/sample-*.json to all use us-west-2 regions as well. What do you have in mind ?